src/topology/continuous_function/algebra.lean

```diff
@@ -825,4 +826,33 @@ instance [has_star R] [has_star β] [has_smul R β] [star_module R β]
 
 end star_structure
 
+variables {X Y Z : Type*} [topological_space X] [topological_space Y] [topological_space Z]
+variables (𝕜 : Type*) [comm_semiring 𝕜]
+variables (A : Type*) [topological_space A] [semiring A] [topological_semiring A] [star_ring A]
+variables [has_continuous_star A] [algebra 𝕜 A]
+
+/-- The functorial map taking `f : C(X, Y)` to `C(Y, A) →⋆ₐ[A] C(X, A)` given by pre-composition
+with the continuous function `f`. See `continuous_map.comp_monoid_hom'` and
+`continuous_map.comp_add_monoid_hom'` for bundlings of pre-composition into a `monoid_hom` and
```

Comment:
I think we should either prime the new definition or unprime the old ones.

Can we blame to find out who named them and why? (I'm on mobile)

Reply:
I'll prime this one for consistency and because there is a post-composition version of this too (but with a `star_alg_hom` instead of  `continuous_map` of course).

Reply:
Ah shoot, I just found some versions living in `topology/continuous_function/compact`, including one for algebra homomorphisms (although that one should probably move to this file).

Reply:
#16875 moves `continuous_map.comp_right_alg_hom` (and generalizes it)